### PR TITLE
Add more owners/approvers/reviewers

### PR DESCRIPTION
**Pull request by liangxia**

### CODE REVIEW

**Review by pruan-rht — COMMENTED** [review]
@liangxia pls add @cpmeadors as `approvers` and `reviewers` category as he's the team lead

**Comment by liangxia** [comment]
> @liangxia pls add @cpmeadors as `approvers` and `reviewers` category as he's the team lead

@pruan-rht 
1. We are not adding all leads. Instead we are adding those team members who are actively in PR reviews.
2. You need to specify which folder/component you are requesting for.
3. Team members are encouraged to request those permissions in separate PRs by themselves so we are sure they are willing to.

File: features/kata/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+approvers:
+  - pruan-rht
```
Comment (pruan-rht) [inline features/kata/OWNERS:2]: @liangxia please add @cpmeadors  as approver

Comment (liangxia) [reply features/kata/OWNERS:3]: Done

**Pull request merged**